projects/laji/src/app/shared-modules/search-filters/three-state-multi-switch/three-state-multi-switch.component.html

```diff
@@ -1,9 +1,9 @@
 <div class="lj-select" [ngClass]="{'lj-close': open, 'lj-open': !open}">
   <strong *ngIf="title !== undefined && title !== ''" class="lj-title link" (click)="toggle($event)">{{ title }}<laji-info *ngIf="info" [html]="info"></laji-info></strong>
   <ng-container *ngIf="options$ | async; let options">
-    <div [hidden]="open">
+    <div [hidden]="open || (options | filterOptions:'id':falseValue:trueValue).length === 0">
```

Comment:
instead of adding this logic to view you could add it in the controller so that open property would reflect the desired value